### ISSUE TITLE
Training word vectors and similarities

### ISSUE THREAD

**Issue by TimoCrafter** [explosion/spaCy#1229]
Just a question:

First an example:

I have a question, `What is the maximum of Email-Accounts I can create?`.

Then I have 2 questions in the FAQ:

`What is the number of E-Mail Accounts I can create?`
and
`What is the maximum of Calendar-Accounts I can create?`.

It seems that the second FAQ is more similar than the first one, but the first is the right one.

So I want to tell SpaCy that the first one is **more similar** than the second one.

In other words: I want to train the word vectors **with examples,** but how to do that?

Kind regards,
Timo

**Comment by honnibal** [comment]
We don't really have an example for this currently. There's an open enhancement issue now, which you can follow here: #2498 . Sorry for not replying to this sooner.
